index.rst

```diff
@@ -511,6 +511,25 @@ particular planning package. Thinking through the plan in this way can help in
 building up a detailed plan in a flexible, agile way, while also ensuring that
 scope, cost and schedule are carefully controlled.
 
+.. _sec-software-release:
+
+Software Releases
+-----------------
+
+Per :ref:`sec-cycle-cadence`, a series of software releases will be made
+throughout LSST construction. These will provide a stable basis upon which
+external users (other subsystems, science collaborations and the wider
+community) can base their work.
+
```

Comment:
Chucks keep mentioning to me every time he seems me that he'd like to see more frequent releases as we are getting closer to commissioning, perhaps even as frequent as monthly. We might start thinking about it, and let's make sure we keep that door open and don't say that we will do releases only once every 6 months.